### DISCUSSION TITLE
when i execute this line :       bot = AgentBot.create!(name: "Rasa Bot", outgoing_url: "http://localhost:8000")

### DISCUSSION THREAD

**Discussion by r-sagar-cs** [chatwoot/chatwoot#2939] [Self hosted]
getting this error: 

ActiveRecord::ConnectionNotEstablished: could not translate host name "postgres" to address: Temporary failure in name resolution

from /usr/local/rvm/gems/ruby-3.0.2/gems/activerecord-6.1.4/lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
Caused by PG::ConnectionBad: could not translate host name "postgres" to address: Temporary failure in name resolution

from /usr/local/rvm/gems/ruby-3.0.2/gems/pg-1.2.3/lib/pg.rb:58:in `initialize'

**Comment by sojan-official** [comment]
Please double-check whether you have the correct environment variables configured for your chatwoot installation. particularly the config for postgres
